templates/includes/sidebar.html

```diff
@@ -0,0 +1,14 @@
+<ul>
+  <li>
+    <a href = "#">Home page</a>
+  </li>
+  <li>
+    <a href = "#">Manufacturers</a>
```

Comment:
The `href` attribute is set to `#`. Update it to use Django's `{% url %}` template tag to link to the Manufacturers page.